### ISSUE TITLE
Issue Occur in LogOut 

### ISSUE THREAD

**Issue by Talhaayubkhan** [hiteshchoudhary/chai-backend#83]
if i send req using postman (generally issue occur in logout route)!!
jwt must be a string
Error: Invalid AccessToken
    at file:///C:/Users/win11/OneDrive/Desktop/Backend/src/middleware/auth.middleware.js:30:17

**Comment by arush73** [comment]
Hey @Talhaayub1 i went through your repo and found out that you the issue 
The issue is in your loginUser function in the User controllers 
in response you are not sending the "tokens in cookies"
it should be - 
`.cookie("accessToken" , accessToken , options)
.cookie("refershToken" , refreshToken , options)`

<img width="390" alt="Screenshot 2024-01-25 130439" src="https://github.com/hiteshchoudhary/chai-backend/assets/116187611/7b686479-99e6-4e58-b281-66b04c7be4f9">

**Comment by Talhaayub1** [comment]
Thank You!

**Comment by Talhaayubkhan** [comment]
yes i notice it, but finally i solve the issue after 2 days!!! Thanks Mannnnn 🙌

**Comment by vrk0ranit** [comment]
I'm getting error in logout user when I send request through Postman it's through an error of jet malformed

**Comment by abhi-20** [comment]
getting same error

![logout_videotube](https://github.com/hiteshchoudhary/chai-backend/assets/58565895/35ef9da6-fee7-4265-bad9-02a4bef856d7)

**Comment by vrk0ranit** [comment]
Hey buddy thanks for addressing my issue.I solved it many days ago this all
happened just because of my typo error

**Comment by Mayank06711** [comment]
Error: jwt must be a string
I`m getting this error on logout request I don't  know how to solve this issue [sourcelink](https://github.com/Mayank06711/backend-project)
checked my code number of times

**Comment by AdarshTheki** [comment]
const options = { httpOnly:true } 
// remove secure: true to resolve this problem to get accessToken to undefine

**Comment by Vedant5125** [comment]
In Postman, the issue with logout occurs because cookies are set in the /login request but are not automatically included in the /logout request.
To test logout properly, reuse the /login request, change the URL to /logout, and then send the request. This ensures the cookies are included, allowing them to be cleared and the user to be logged out successfully.
